test/Microsoft.ML.CpuMath.UnitTests.netcoreapp/UnitTests.cs

```diff
@@ -271,7 +272,7 @@ public void AddScalarUTest(string mode, string test, string scale, Dictionary<st
             RemoteExecutor.RemoteInvoke((arg0, arg1, arg2) =>
             {
                 CheckProperFlag(arg0);
-                float defaultScale = float.Parse(arg2);
+                float defaultScale = float.Parse(arg2, NumberStyles.Any, CultureInfo.InvariantCulture);
```

Comment:
does it work to just use

```C#
float.Parse(arg2, CultureInfo.InvariantCulture);
```

?

Reply:
Ugh, missed that particular overload. Will update.